scripts/Muon/GUI/Common/fitting_tab_widget/fitting_tab_model.py

```diff
@@ -196,30 +195,53 @@ def add_fit_to_context(self, parameter_workspace, function,
             parameter_workspace, self.function_name,
             input_workspace, output_workspace_name, global_parameters)
 
+    def change_plot_guess(self, plot_guess, parameter_dict):
+        try:
+            fit_function = parameter_dict['Function']
+            data_ws_name = parameter_dict['InputWorkspace']
+        except KeyError:
+            return
+        guess_ws_name = '{}_guess'.format(data_ws_name)
```

Comment:
Can we make these guess workspaces hidden workspaces so as to not clutter up the ADS and remove old guess workspaces when we create a new one? (any workspace whose name starts with a double underscore will be hidden)

Or would it be possible to not add them to the ADS at all ?